bio_hansel/main.py

```diff
@@ -12,10 +12,12 @@
 import pandas as pd
 from collections import defaultdict
 
-from . import program_name, program_desc, __version__
-from .subtyper import subtype_fasta, SUBTYPE_SUMMARY_COLS, subtype_reads
-from .subtype_stats import subtype_counts
-from .utils import genome_name_from_fasta_path, get_scheme_fasta
+from bio_hansel import program_name, program_desc, __version__
```

Comment:
Please change all imports to `from . import ...` format.

Reply:
It seems when I leave it at `..` instead of `bio_hansel.` PyCharm won't let me run the program. Perhaps this is an error with my setup?